doc/node_config_json_schema.json

```diff
@@ -0,0 +1,609 @@
+      "required": []
+    },
+    "inspect-brk": {
+      "type": "boolean",
+      "required": []
+    },
+    "inspect-brk-node": {
+      "type": "boolean",
+      "required": []
+    },
+    "inspect-wait": {
+      "type": "boolean",
+      "required": []
+    },
+    "inspect-publish-uid": {
+      "type": "string",
+      "required": []
+    },
+    "report-exclude-network": {
+      "type": "boolean",
+      "required": []
+    },
+    "report-exclude-env": {
+      "type": "boolean",
+      "required": []
+    },
+    "tls-max-v1.2": {
+      "type": "boolean",
+      "required": []
+    },
+    "tls-min-v1.3": {
+      "type": "boolean",
+      "required": []
+    },
+    "tls-min-v1.2": {
+      "type": "boolean",
+      "required": []
+    },
+    "tls-min-v1.1": {
+      "type": "boolean",
+      "required": []
+    },
+    "tls-min-v1.0": {
+      "type": "boolean",
+      "required": []
+    },
+    "tls-keylog": {
+      "type": "string",
+      "required": []
+    },
+    "interactive": {
+      "type": "boolean",
+      "required": []
+    },
+    "import": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "required": []
+      }
+    },
+    "print": {
+      "type": "boolean",
+      "required": []
+    },
+    "watch": {
+      "type": "boolean",
+      "required": []
+    },
+    "unhandled-rejections": {
+      "type": "string",
+      "required": []
+    },
+    "trace-require-module": {
+      "type": "string",
+      "required": []
+    },
+    "trace-env-native-stack": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-env": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-promises": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-warnings": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-uncaught": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-tls": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-sync-io": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-exit": {
+      "type": "boolean",
+      "required": []
+    },
+    "throw-deprecation": {
+      "type": "boolean",
+      "required": []
+    },
+    "trace-env-js-stack": {
+      "type": "boolean",
+      "required": []
+    },
+    "test-udp-no-try-send": {
+      "type": "boolean",
+      "required": []
+    },
+    "test-coverage-exclude": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "required": []
+      }
+    },
+    "test-skip-pattern": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "required": []
+      }
+    },
+    "test-only": {
+      "type": "boolean",
+      "required": []
+    },
+    "test-reporter-destination": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "required": []
+      }
+    },
+    "experimental-test-snapshots": {
+      "type": "null",
```

Comment:
How can this be null? I'm not following

Reply:
The flag is a no-op, we get the value 0 from the env flag parser which means no-op

Reply:
Can we remove them from this file?

Reply:
yes

Reply:
I removed them for now but not sure if we want to keep them or not